packages/network-console-shared/src/file_io/native/test/environment-container-adapter.test.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// tslint:disable:no-unused-expression

```

Comment:
why?

Reply:
The tests frequently include this because Chai assertions can be expressions, e.g.

    expect(someExpr).to.be.true;